src/transform/convert/generateControlFields.js

```diff
@@ -27,8 +28,10 @@ export function handle007(marcRecord, record) {
   }
 }
 
-export function handle008(marcRecord, moment) {
```

Comment:
Moment has to come from parent function.
This way we can give certain date to tests so that they are not changing every day.

Date we are giving to tests is 01.01.2000

https://github.com/NatLibFi/melinda-record-import-transformer-helmet/blob/9d815b5ec649b02f4bffce0585851a5d4086740b/src/transform/index.spec.js#L62